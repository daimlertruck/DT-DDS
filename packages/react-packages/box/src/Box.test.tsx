import { render } from '@testing-library/react';

import { Box } from '.';

describe('<Box /> component', () => {
  it('should render a span html element with the given content', () => {
    const { container } = render(
      <Box element='span'>
        <p>Some Content</p>
        <p>More Content</p>
      </Box>
    );

    expect(container).toMatchSnapshot();
  });
});
