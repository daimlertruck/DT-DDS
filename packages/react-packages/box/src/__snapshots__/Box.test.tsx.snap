// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> component should render with default div element and children 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div>
  <div
    class="emotion-0 emotion-1"
  >
    <p>
      Some Content
    </p>
    <p>
      More Content
    </p>
  </div>
</div>
`;
